tap_pipedrive/tap.py

```diff
@@ -244,17 +244,43 @@ def execute_stream_request(self, stream):
         return self.execute_request(stream.endpoint, params=params)
 
     def execute_request(self, endpoint, params=None):
-        headers = {"User-Agent": self.config["user-agent"]}
-        _params = {
-            "api_token": self.config["api_token"],
-        }
-        if params:
-            _params.update(params)
-
         url = "{}/{}".format(BASE_URL, endpoint)
-        logger.debug("Firing request at {} with params: {}".format(url, _params))
+        logger.debug("Firing request at {} with params: {}".format(url, params))
 
-        return requests.get(url, headers=headers, params=_params)
+        # remove this case when all customers have authed with oauth
```

Comment:
if you use `# TODO: ` then vscode has tooling to pick this up for you :)

Reply:
👍